util/src/test/java/com/netcracker/ConsoleHelperTest.java

```diff
@@ -0,0 +1,89 @@
+package com.netcracker;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.TextFromStandardInputStream;
+import org.junit.rules.ExpectedException;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.contrib.java.lang.system.TextFromStandardInputStream.emptyStandardInputStream;
+
+public class ConsoleHelperTest {
+    private static final PrintStream SYSTEM_OUT = System.out;
+    @Rule
+    public final TextFromStandardInputStream systemIn = emptyStandardInputStream();
```

Comment:
I don't understand the part you are saying about "it" being a constant. What is a constant ?
I understand that you are switching standard out stream with your custom one to make assertions on what comes out. But you can do it in a more elegant way, using a @Rule for output streams.